guides/common/modules/proc_deleting-audit-records-manually.adoc

```diff
@@ -0,0 +1,21 @@
+[id="deleting-audit-records-manually_{context}"]
+= Deleting audit records manually
+
+You can use the `foreman-rake audits:expire` command to remove audit records at any time.
+
+.Procedure
+* Delete the audit records using the `foreman-rake audits:expire` command:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# foreman-rake audits:expire days=_Number_Of_Days_
+----
++
+This command deletes all audit records older than `_Number_Of_Days_`.
++
+[NOTE]
+====
+Without the `days` argument, the `foreman-rake audits:expire` command removes audit records that are older than the number of days specified by the `Saved audits interval` setting.
+If this setting has a value set, this deletion is done automatically and does not need to be performed manually.
+The command without the `days` argument does nothing if the `Saved audits interval` does not have a value set.
+====
```

Comment:
After poking around further:
* The output of `foreman-rake audits:expire` seems to suggest that it does nothing. So no need state that explicitly here. (Granted, the output could be clearer... but that's a different story, one that should be fixed on the product side itself if it turns out to be an issue for anyone.)
* Sometimes it's nice to point to a man page or --help page in `.Additional resources` to provide this kind of detail about command behavior. But that doesn't seem to be an option in case of this command.